source/includes/rankings.md

```diff
@@ -216,3 +217,568 @@ Header            | Description
+
+# This header turns on compression to reduce the bandwidth usage and transfer time.
+headers = {'Accept-Encoding': 'deflate, gzip'}
+
+# Python Main Code Sample
+response = requests.get(request_url,
+                        auth=(username, password),
+                        params=req_params,
+                        headers=headers,
+                        stream=True)
+
+print response.status_code
+for line in response.iter_lines():
+  # Load json object and print it out
+  json_record = json.loads(line)
+  print json_record
+```
+
+```java
+// This example uses java Unirest library http://unirest.io/java.html
+
+// Request Parameters
+store = "android";      // Could be either "android" or "itunes".
+date = "{{ date }}";    // Date in YYYY-MM-DD format.
+
+// Auth Parameters
+username = "{API_KEY}"; // Replace {API_KEY} with your own API key.
+password = "X";         // Password can be anything.
+
+// Request URL
+requestUrl = "https://api.appmonsta.com/v1/stores/" + store + "/rankings/genres.json";
+
+// Java Main Code Sample
+HttpResponse response = Unirest.get(requestUrl)
+  // This header turns on compression to reduce the bandwidth usage and transfer time.
+  .header("Accept-Encoding", "deflate, gzip")
+  .basicAuth(username, password)
+  .queryString("country", $countryCode),
+  .queryString("date", $date),
+  .asString();
+
+int status = response.getStatus();
+
+BufferedReader in = new BufferedReader(new InputStreamReader(response.getRawBody()));
+String line = null;
+while((line = in.readLine()) != null) {
+  System.out.println(line);
+}
+```
+
+```php
+<?php
+// Request Parameters
+$store = "android";      // Could be either "android" or "itunes".
+$date = "{{ date }}";    // Date in YYYY-MM-DD format.
+
+// Auth Parameters
+$username = "{API_KEY}"; // Replace {API_KEY} with your own API key.
+$password = "X";         // Password can be anything.
+
+// Request URL
+$url = "https://api.appmonsta.com/v1/stores/$store/rankings/genres.json?date=$date";
+
+// PHP Main Code Sample
+$ch = curl_init();
+curl_setopt($ch, CURLOPT_URL,$url);
+curl_setopt($ch, CURLOPT_USERPWD, $username . ":" . $password);
+curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+curl_setopt($ch, CURLOPT_CONNECTTIMEOUT, 500);
+curl_setopt($ch, CURLOPT_WRITEFUNCTION, function($curl, $data) {
+    // Load json object and print it out
+    $json_record = json_decode((string)$data, true);
+    echo json_encode($json_record);
+    return strlen($data);
+});
+curl_exec($ch);
+curl_close($ch);
+?>
+```
+> The above code loads one JSON per line and prints it out:
+
+```json
+{"name":"Music & Audio","genre_id":"MUSIC_AND_AUDIO"}
+{"name":"Auto & Vehicles","genre_id":"AUTO_AND_VEHICLES"}
+{"parent_id":"FAMILY","name":"Family Brain Games","genre_id":"FAMILY_BRAINGAMES"}
+
+{"name":"Books","genre_id":"6018"}
+{"name":"Stickers","genre_id":"6025"}
+{"parent_id":"6021","name":"Health, Mind & Body","genre_id":"13017"}
+```
+
+Request all ranking genres by date.
```

Comment:
Request all available ranking genres (also know as categories) by date. If you use (link to aggregate API call) then you this call will map genre IDs to human readable genre names.